gardener/kube-apiserver.yaml

```diff
@@ -18,11 +18,10 @@ spec:
   interval: 1m
   chart:
     spec:
-      chart: garden-kube-apiserver
-      version: 4.2.0
+      chart: ./helmcharts/garden-kube-apiserver
       sourceRef:
-        kind: HelmRepository
-        name: gardener-community-charts
+        kind: GitRepository
+        name: yake
```

Comment:
@timebertt I moved `23ke` to `yake`. 

According to new https://yake.cloud/release-notes/v1.86/